### DISCUSSION TITLE
Failed to create database webhook - permission denied for schema supabase_functions

### DISCUSSION THREAD

**Discussion by wes-goulet** [supabase/supabase#19476] [Questions]
# Bug report

## Describe the bug

I cannot create a DB webhook for my hosted projected.

> This seems similar to #12008 but that issue has been marked fixed.

## To Reproduce

1. On Supabase dashboard go to Database -> Webhooks
2. Click on "Enable Webhooks" (note: after enabling if I leave the page and come back the "Enable webhooks" button is back, it's like it doesn't remember I already enabled them)
3. Fill in the webhook info - name: "test", table: [some table from public schema], Events: "INSERT", type: HTTP Request, GET, URL: [some valid URL]
4. Click "Confirm"
5. I get an error

```text
Failed to create hook: failed to create pg.triggers: permission denied for schema supabase_functions
```

## Expected behavior

A new webhook is created.

## Screenshots

![image](https://user-images.githubusercontent.com/3293318/219080176-92664ca2-f9a6-4c83-93da-aca8b07285ab.png)

## System information

- OS: macOS
- Browser (if applies): Edge 108.0.1462.54 (ARM)
- Version of supabase-js: NA
- Version of Node.js: NA
- Hosted instance PG_NET version: 0.7.1

## Additional context

I do have a migration script that alters the table owner to "postgres" (because I ran into [this](https://supabase.com/docs/guides/cli/managing-environments#permission-denied-on-db-push)), not sure if that is related.

I just paused/unpaused my free instance this morning so I should have the latest image (I wanted to make sure I had the fix for https://github.com/supabase/pg_net/issues/82).

I tried manually turning on the PG_NET extension first (which I was able to manually turn on once I had the fix for https://github.com/supabase/pg_net/issues/82) and then seeing if I could make a webhook then but I get the same error.

If I run the following SQL `select net.check_worker_is_up()` I see the result `Failed to run sql query: the pg_net background worker is not up`

**Comment by eleddie** [comment]
I'm getting a similar issue when running supabase locally. 
I get the following error:
```bash
Error: ERROR: schema "supabase_functions" does not exist (SQLSTATE 3F000)
```
Also happening when running the test in the github actions from this documentation:

https://supabase.com/docs/guides/cli/managing-environments#configure-github-actions
(The ci.yml file)

**Comment by kevinmlong** [comment]
I'm having a similar issue with my local hosted supabase using docker (not supabase cli). Has anyone found a solution or workaround to this?

**Comment by wes-goulet** [comment]
I ended up working around this.  First I made a new project on Supabase and the webhook setup worked this time.  But I still wanted to test webhooks locally using the CLI, and I could never get that to work.

So I worked around it by making my own webhook function.  I [wrote down how I did it](https://goulet.dev/posts/supabase-webhook-local/) in case anyone who stumbles upon this is interested.
